js/tsconfig.json

```diff
@@ -12,7 +12,7 @@
     "esModuleInterop": true,
     "strict": true,
     "target": "es2022",
-    "lib": ["es2022"],
+    "lib": ["es2022", "DOM"],
```

Comment:
I wonder if you only need to make this change for `js/genkit` package, not everything...

Reply:
I don't know, @inlined made the change on js/clients and it doesn't compile on my machine. I just tried replicating that on the parent directory (`js`) and it worked...